### ISSUE TITLE
Condense inverted data into reasonable txt file for LLM

### ISSUE THREAD

**Issue by RuruHoong** [RuruHoong/AIjobsearch#7]

**Comment by RuruHoong** [comment]
@ethanmatlin Let me know if you're able to look into this before our next meeting? For context, this is the sample I ended up adding into Poe with each ID as a row, and I think that worked best out of all the iterations. 
[output.txt](https://github.com/RuruHoong/AIjobsearch/files/14230309/output.txt)

**Comment by ethanmatlin** [comment]
Uploaded to Dropbox! In Path Dependency in Job Choice Example / Output / new* (all of the new format files start with "new"). They are numbered 1-16. Much smaller now-- Only 261 KB total so this seems like it'll be no problem with the upload restrictions hopefully!

**Comment by RuruHoong** [comment]
Thanks so much!! Will try this :)

**Comment by RuruHoong** [comment]
Hmm that's funny, there's only about 30-40 observations per file, but surely there are more than 40*16 alumni?

**Comment by RuruHoong** [comment]
For future self: csv_to_txt.R in Output converts these csvs into text for input into Poe.

**Comment by ethanmatlin** [comment]
> Hmm that's funny, there's only about 30-40 observations per file, but surely there are more than 40*16 alumni?

Yup, you're right -- sorry, there was a bug. Fixed and should have the new ones ready by the early afternoon

**Comment by RuruHoong** [comment]
- [ ] @RuruHoong to convert to txt and upload to Poe by next meeting
